src/ensembl/src/shared/components/help-popup/useHelpArticle.ts

```diff
@@ -72,7 +72,7 @@ const useHelpArticle = (reference: ArticleReference | VideoReference) => {
 
   const query = reference.type === 'article' ? getQuery(reference) : null;
   const { helpApiHost } = config;
```

Comment:
If you deleted `helpApiHost` from the config, the import should be deleted from here as well